WcaOnRails/spec/mailers/previews/competitions_mailer_preview.rb

```diff
@@ -1,16 +1,18 @@
 # Preview all emails at http://localhost:3000/rails/mailers/competitions_mailer
 class CompetitionsMailerPreview < ActionMailer::Preview
-
-  # Preview this email at http://localhost:3000/rails/mailers/competitions_mailer/notify_board_of_confirmed_competition
   def notify_board_of_confirmed_competition
     c = CompetitionDelegate.first.competition
     CompetitionsMailer.notify_board_of_confirmed_competition(c.delegates[0], c)
   end
 
-  # Preview this email at http://localhost:3000/rails/mailers/competitions_mailer/notify_users_of_results_presence
   def notify_users_of_results_presence
     competition = Competition.joins(:results).where.not(results_posted_at: nil).first
     user = competition.competitor_users.first
     CompetitionsMailer.notify_users_of_results_presence(user, competition)
   end
+
+  def submit_results_nag
+    competition = Competition.first
```

Comment:
I just thought `.first` can't be right.

Now I realize this is about looking at a sample version of the email, so grabbing a random competition to show actually _can_ be the right thing. Taking the newest would give it more variety I guess.

Long story short, I don't know anything about this.